model/terms_of_service_test.go

```diff
@@ -8,44 +8,31 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 )
 
 func TestTermsOfServiceIsValid(t *testing.T) {
 	s := TermsOfService{}
 
-	if err := s.IsValid(); err == nil {
-		t.Fatal("should be invalid")
-	}
+	require.Error(t, s.IsValid(), "should be invalid")
```

Comment:
Same comment as in https://github.com/mattermost/mattermost-server/pull/12579: Please use `assert` here.